BREEZE.rst

```diff
@@ -603,50 +588,38 @@ your local sources into the ``/opt/airflow`` location of the sources within the
     :align: center
     :alt: Source code mapping
 
+Breeze Command-Line Interface Reference
+=======================================
 
-Airflow Breeze flags
-====================
+Airflow Breeze Syntax
+---------------------
 
-These are the current flags of the `./breeze <./breeze>`_ script
+This is the current syntax for  `./breeze <./breeze>`_:
```

Comment:
FYI: this output is copy&pasted directly from ./breeze --help, so we will need to synchronize it with https://github.com/PolideaInternal/airflow/blob/gsod-breeze-documentation-update/breeze where it is generated from. I will do it later before we merge so no worries about it for now - let's update it in the docs and I will put it back to the help script.

Reply:
ok, thanks!